tests/PHPStan/Analyser/data/bug-4896.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Bug4896;
+
+/** @var \DateTime|\DateInterval $command */
+$command = new \DateTime();
+
+switch ($command::class) {
+	case \DateTime::class:
+		var_dump($command->getTimestamp());
```

Comment:
These test-cases would be much more useful implemented in NodeScopeResolverTest, thanks.

Reply:
Done. :relaxed: